docs/plugins/assign.md

```diff
@@ -0,0 +1,31 @@
+# assign
+
+## 设计背景
+
+通过命令让机器人来处理用户指派的事务，同时可以添加一些说明来明确指派原因。
+
+## 设计思路
+
+该插件提供了两个命令，只需要在Issue添加评论即可触发：
```

Comment:
I think also work for PR.

In addition, space is needed between Chinese and English.